bumble/transport/android_emulator.py

```diff
@@ -33,7 +33,7 @@
 
 
 # -----------------------------------------------------------------------------
-async def open_android_emulator_transport(spec):
+async def open_android_emulator_transport(spec: str | None) -> PumpedTransport:
```

Comment:
suggest: returning just `Transport` to allow the internal type to change in the future without affecting callers.

Reply:
Done